src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```
Static methods are in general a bad practice. You should create an instance of a class which method you want to call.